Server/src/controllers/authenticationController.ts

```diff
@@ -122,8 +122,9 @@ export class AuthenticationController {
         let serviceResponse: IResponse
         try {
             serviceResponse = await this.authenticationService.checkLoginCredentials(LoginInfo);
-            response.setHeader('Set-Cookie', serviceResponse.message);
-            return response.status(serviceResponse.statusCode).json(serviceResponse.message);
+            response.cookie('token', serviceResponse.message, { httpOnly: true })
```

Comment:
We will need to add these flags for prod and https: 
-  HttpOnly, Secure flag and SameSite=Lax|Strict

Reply:
done